tests/test_apply.py

```diff
@@ -22,6 +22,7 @@
 
 import numpy as np
 import pennylane as qml
+from pennylane.ops.qubit import ControlledPhaseShift
```

Comment:
```suggestion
```

This line isn't needed.

Reply:
It's added automatically. I found it strange too, but, thought that might be part of the code standardization. I ensure that'll be removed. Thanks!